include/sys/syscall_lookup.h

```diff
@@ -251,6 +251,7 @@ SYSCALL_LOOKUP(futimens,                   2)
 SYSCALL_LOOKUP(munmap,                     2)
 
 #if defined(CONFIG_PSEUDOFS_SOFTLINKS)
+  SYSCALL_LOOKUP(link,                     2)
```

Comment:
Maybe this can be a libc call instead of syscall?

Reply:
We plan to implement fs hard link and then link will have a different implementation compared with symlink, so it's better to keep link as a syscall.